distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc

```diff
@@ -0,0 +1,33 @@
+.[[org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin]]Expiration Date Pre-Ingest Plugin
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Overwrite If Empty
+|overwriteIfBlank
+|Boolean
+|If this is checked, overwrite all 'blank' expiration dates in metacards. If this is not checked, leave metacards with 'blank' expiration dates as-is.
+|false
+|true
+
+|Overwrite If Exists
+|overwriteIfExists
+|Boolean
+|If this is checked, overwrite all 'existing' non-empty expiration dates in metacards with a new date. If this is not checked, leave metacards with an existing expiration date.
+|false
+|true
+
+|Offset from Created Date (in days)
+|offsetFromCreatedDate
+|Integer
+|A metacard's new expiration date is calculated by adding this value (in days) to its created date.
+|true
```

Comment:
❗️ Shouldn't those be the other way around, i.e., `Default Value` = `30` and `Required` = `true`?